### PR TITLE
DDF-2803 Fixing the simple search UI by adding an import package

### PR DESCRIPTION
**Pull request by brjeter**
#### What does this PR do?
Simple Search UI was missing an import package causing the webapp to fail to come up despite the bundle coming up fine.
#### Who is reviewing it? 
@mweser @emmberk @mackncheesiest 
#### Select relevant component teams: 
https://github.com/orgs/codice/teams
#### Choose 2 committers to review/merge the PR.
@bdeining 
@lessarderic 
#### How should this be tested? (List steps with links to updated documentation)
Build and install DDF. Verify the Simple Search UI loads.
#### What are the relevant tickets?
[DDF-2803](https://codice.atlassian.net/browse/DDF-2803)
#### Checklist:
- [] Documentation Updated
- [] Update / Add Unit Tests
- [] Update / Add Integration Tests
#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

**Comment by mweser** [comment]
I will manually test to verify that gzipping is working in simple search ui.

**Comment by mackncheesiest** [comment]
I'm heroing atm. Just finished quick build.

**Comment by mackncheesiest** [comment]
Verified Simple Search UI loads. Not seeing the `Content-Encoding: gzip` response header for elements on the page, however.
image

File: catalog/ui/search-ui/simple/pom.xml [hunk 1]
```diff
@@ -133,7 +133,8 @@
                             javax.servlet,
                             org.osgi.framework,
                             org.slf4j,
-                            javax.servlet.http
+                            javax.servlet.http,
```
Comment (mweser) [inline catalog/ui/search-ui/simple/pom.xml]: ❗️ This does not fix issue. Mirror this: https://github.com/mweser/ddf/commit/a25233cb8530fcb00ad2d59a7111c1b36e9dc75c

Comment (mweser) [inline catalog/ui/search-ui/simple/pom.xml]: screen shot 2017-08-23 at 11 44 15

**Comment by brjeter** [comment]
Committed another change in response to @mweser's comment. It removes the Import-Package declaration entirely to fallback to the default behavior of importing everything it needs. This fixes the issue of the UI coming up and fixes the Gzip filtering. It also aligns more with our best practices for bundles.

**Comment by brjeter** [comment]
build now

**Comment by mweser** [comment]
Manual testing with latest changes successful. 
screen shot 2017-08-23 at 12 37 57

**Pull request merged**